cmd/trace-agent/config/config.go

```diff
@@ -222,6 +221,16 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 	if coreconfig.Datadog.IsSet("apm_config.max_remote_traces_per_second") {
 		c.MaxRemoteTPS = coreconfig.Datadog.GetFloat64("apm_config.max_remote_traces_per_second")
 	}
+	if k := "apm_config.features"; coreconfig.Datadog.IsSet(k) {
```

Comment:
Would `"apm_config.features"` be better as a constant?

Reply:
@knusbaum , how do you mean? Like instead of `if k := "apm_config.features"; coreconfig.Datadog.IsSet(k)....`, having something like
`const apmFeatures = "apm_config.features"
{... other stuff ... }
if coreconfig.Datadog.IsSet(apmFeatures)...
`
If yes^, then this begs the question: Should the rest of the strings below be constants as well? E.g, "apm_config.ignore_resources" in `if k := "apm_config.ignore_resources"; coreconfig.Datadog.IsSet(k)`, and many others below.

Reply:
Hmm, yeah I see the below are done the same way.

I think they all *would* be better as constants, since they are used in various places in the code, but that's not something that needs to be done in this PR.

We can leave it as it is to remain consistent with what's already here.